pkg/services/sqlstore/dashboard_snapshot_test.go

```diff
@@ -5,169 +5,169 @@ import (
 	"testing"
 	"time"
 
-	"github.com/grafana/grafana/pkg/util"
-
-	. "github.com/smartystreets/goconvey/convey"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	"github.com/grafana/grafana/pkg/components/simplejson"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/setting"
 )
 
 func TestDashboardSnapshotDBAccess(t *testing.T) {
-	Convey("Testing DashboardSnapshot data access", t, func() {
-		InitTestDB(t)
+	t.Helper()
+
+	InitTestDB(t)
 
-		origSecret := setting.SecretKey
-		setting.SecretKey = "dashboard_snapshot_testing"
-		t.Cleanup(func() {
-			setting.SecretKey = origSecret
+	origSecret := setting.SecretKey
+	setting.SecretKey = "dashboard_snapshot_testing"
+	t.Cleanup(func() {
+		setting.SecretKey = origSecret
+	})
+
+	t.Run("Given saved snapshot", func(t *testing.T) {
+		cmd := models.CreateDashboardSnapshotCommand{
+			Key: "hej",
+			Dashboard: simplejson.NewFromAny(map[string]interface{}{
+				"hello": "mupp",
+			}),
+			UserId: 1000,
+			OrgId:  1,
+		}
+		err := CreateDashboardSnapshot(&cmd)
+		require.NoError(t, err)
+
+		t.Run("Should be able to get snapshot by key", func(t *testing.T) {
+			query := models.GetDashboardSnapshotQuery{Key: "hej"}
+			err = GetDashboardSnapshot(&query)
+			require.NoError(t, err)
+
+			assert.NotNil(t, query.Result)
+
+			dashboard, err := query.Result.DashboardJSON()
+			require.NoError(t, err)
+			assert.Equal(t, dashboard.Get("hello").MustString(), "mupp")
```

Comment:
assert.Equal takes the wanted value before the actual one.
```suggestion
			assert.Equal(t, "mupp", dashboard.Get("hello").MustString())
```